apps/test/unit/applab/ChartApiTest.js

```diff
@@ -1,5 +1,4 @@
 'use strict';
-/* global describe, beforeEach, it, Promise */
```

Comment:
These are [implicitly provided by the ESLint "mocha" environment](https://github.com/code-dot-org/code-dot-org/blob/staging/.eslintrc.js#L10).